### PR TITLE
spicing up rewording and padding fun fact count

### PR DESCRIPTION
**Pull request by RayDeeUx**

### CODE REVIEW

File: docs/assets/funfacts.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Touching grass is not needed to complete skyblock, dont listen to those idiots
 Floors 8 through 10 of the Catacombs were originally planned for release,\nbut portions of it were later merged into Floor 7 and its Master Mode counterpart.
 SkyClient is not a client, it's an installer for Forge and mods + packs.
 At one point, Feather Client was willing to pay $300 USD to add Skytils.
-SkyblockPersonalized was once (removed now) the cause of countless rickrolls\nfrom right-clicking any of the Mathematics Farming Hoes.
+SkyblockPersonalized was once the cause of countless rickrolls from right-clicking\nany of the Mathematics Farming Hoes. (It's been removed by now, of course.)
```
Comment (KTibow) [inline docs/assets/funfacts.txt:7]: would this be clipped off

File: docs/assets/funfacts.txt [hunk 2]
```diff
@@ -17,11 +17,17 @@ SkyClient is for more than Skyblock, even though it started out that way.
...
 People asking you to "test their mod" are always trying to rat you.\nThe only ones you can trust are ones widely trusted by the community.
 SBE bad lol
 Here's a good list of ways to stop yourself from getting ratted:\nhttps://bit.ly/HySbModSafetySCC
 Have a great day!
+Don't be scared to ask for support! SkyClient's Discord server often has over 20 concurrent support tickets open at any given day.\n\nOkay, maybe you should be mildly scared.
+#denied-mod-suggestions is an informal guideline for what mods or packs to suggest for SkyClient. Please read it!
+Feel like you're still missing a few mods? No worries-just head over to #bot-commands on the Discord server and drop the command named /modlist.
+Opal is a hydrated amorphous form of silica (SiO2·nH2O); its water content may range from 3 to 21% by weight, but is usually between 6 and 10%.
+Depending on the conditions in which it formed, opal may be transparent, translucent, or opaque, and the background color may be white, black, or nearly any color of the visual spectrum.
+If you can read this, you are definitely not on macOS. Otherwise, all you'll see is a Mojang logo.
```
Comment (KTibow) [inline docs/assets/funfacts.txt:33]: we need more people's opinions, also i feel like a lot of these would be clipped
@nacrt @UnderscoreRayless what do you think

**Review by Zetvue — APPROVED** [review]

**Comment by RayDeeUx** [comment]
I did remember allowing edits from maintainers but sure

**Comment by RayDeeUx** [comment]
@KTibow cmon at least keep the opan facts there

let the obnoxious salmon opal lore remain to some extent

**Comment by KTibow** [comment]
> obnoxious salmon opal lore

????
can you @mention salmon

**Pull request merged**